_config.yml

```diff
@@ -43,10 +43,18 @@ defaults:
       type: events
     values:
       layout: event
+  - scope:
+      path: ""
+      type: resources
+    values:
+      layout: resource
 
 collections:
   events:
     output: true
     permalink: /:year-:month-:day.html
   posts:
     permalink: /:year-:month-:day-:title.html
+  resources:
+    output: true
+    permalink: /:title.html
```

Comment:
Ah, I'm learning about Jekyll. So adding a `/` at the end of a permalink creates a folder with an index.html inside of it? That does seem like overkill.

it seems the GitHub Pages server is configured to work with or without the .html extension. For example both of these already work:

https://musichackathon.github.io/mmh_jekyll/2017-02-18
https://musichackathon.github.io/mmh_jekyll/2017-02-18.html

...so I think it's just a matter of which one we want to link to and want search engines to index. And probably better to leave out the .html extension?

I'd be happy to address permalinks in a separate PR. If you wanna squash and/or merge this it seems good to me!